proto/rill/runtime/v1/api.proto

```diff
@@ -657,7 +657,21 @@ message GetModelSplitsResponse {
 }
 
 message CreateTriggerRequest {
+  // Instance to target.
   string instance_id = 1;
+  // Resources to trigger. See RefreshTriggerSpec for details.
+  repeated ResourceName resources = 4;
+  // Models to trigger. Unlike resources, this supports advanced configuration of the refresh trigger.
+  repeated RefreshModelTrigger models = 5;
+  // Parser is a convenience flag to trigger the global project parser.
+  // Triggering the project parser ensures a pull of the repository and a full parse of all files.
+  bool parser = 6;
+  // Convenience flag to trigger all sources and models.
+  bool all_sources_models = 7;
+  // Convenience flag to trigger all sources and models.
+  // Will trigger models with RefreshModelTrigger.full set to true.
+  bool all_sources_models_full = 8;
+  // DEPRECATED (2024-08-28): Kept for backwards compatibility.
```

Comment:
Why is the backwards compatibility required ?
As long as admin is compatible with CLI it should be enough ?